Source/WebCore/dom/Document.h

```diff
@@ -1680,7 +1680,7 @@ class Document
     void resetObservationSizeForContainIntrinsicSize(Element&);
 
     RefPtr<ViewTransition> startViewTransition(RefPtr<ViewTransitionUpdateCallback>&& = nullptr);
-    ViewTransition* activeViewTransition() const;
+    WEBCORE_EXPORT ViewTransition* activeViewTransition() const;
```

Comment:
Maybe add `hasActiveViewTransition()` and export that instead, like `Page` does.